src/com/main/db/Database.java

```diff
@@ -317,23 +359,44 @@ public void addOrderItem(Order order, Item item) {
 		
 		dbClose();
 	}
-
-	public boolean validateVendor(String username,String password) {
```

Comment:
What was this and where was the rest of it

Reply:
Moved up to vendor section of Database.java